.github/ISSUE_TEMPLATE/⚙️-submit-a-new-resource.md

```diff
@@ -1,25 +1,20 @@
 ---
+order: 3
```

Comment:
Just because it's hard to visualize, is the plan to have templates for:
1. submitting an idea
2. submitting a template to add to the gallery
3. submitting a contribution to the readme

Reply:
yes! those are the 3 that are more specific to awesome-azd. And then I kept bug report and feature request which would be targeted at the website itself. So, it would look like

1. submitting an idea
2. submitting a template
3. submitting a readme contribution
4. bug report 
5. feature request